src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -126,21 +143,21 @@
                   }}
                   selectedValue={pickerHour}>
                   {hour.map((h, index) => (
-                    <Picker.Item key={index} label={h} value={h} />
+                    <Picker.Item key={index} label={h} value={h} color={COLORS.white} />
                   ))}
                 </Picker>
               </View>
             </View>
             <View style={S.pickercontainer}>
-              <Text style={S.pickerlabeltext}>Min</Text>
+              <Text style={S.pickerlabeltext}>Minutes</Text>
               <View style={{}}>
                 <Picker
                   onValueChange={(itemValue) => {
                     setPickerMinute(itemValue);
                   }}
                   selectedValue={pickerMinute}>
                   {minute.map((m, index) => (
-                    <Picker.Item key={index} label={m} value={m} />
+                    <Picker.Item key={index} label={m} value={m} color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `color` prop of the `Picker.Item` components is being set to `COLORS.white`. This is a good practice as it ensures consistency in the UI design.


<!-- This is an auto-generated comment by OSS CodeRabbit -->